### DISCUSSION TITLE
Are Auth collection username , email system fields constrained?

### DISCUSSION THREAD

**Discussion by pnmcosta** [pocketbase/pocketbase#2268] [Q&A] [answered]
I could not find a specific answer to this one, are auth collection system fields **username** and **email** constrained by default?

**Comment by ganigeorgiev** [comment]
Could you elaborate what do you mean by "constrained"?

**Reply by pnmcosta** [reply]
Unique per collection?

I have a use case where a "member" could belong to multiple "users" (both auth collections) and whether I should use a multiple relation:

![image](https://user-images.githubusercontent.com/550684/231235644-09772a01-9f74-4457-9d8e-887d48f67228.png)

Or have duplicates with a single relation, if these fields are not uniquely constrained by default.

**Reply by ganigeorgiev** [reply, marked as answer]
I'm still not sure that I understand your use case and what is the purpose of the attached screenshot.

If you are asking whether the `username` and `email` have a system `UNIQUE` index constraint per collection, then Yes and it can't be changed (_for the `email` it is partial and valid only for the non-empty values because depending on your auth collection options it could be optional_).

**Reply by pnmcosta** [reply]
Thank you, that clears it up. I'll use a collection in the middle instead.
